examples/ts-without-babel/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "target": "es2018",
+    "module": "esNext",
+    "moduleResolution": "node",
+    "lib": ["dom", "es2015", "es2016", "es2017", "es2018", "es2019", "esnext"],
+    "sourceMap": true,
+    "jsx": "react",
+    "baseUrl": ".",
```

Comment:
I don't think we need to use `baseUrl`?

Reply:
It's for direct `import Some from 'src/some.ts'`. I think it's a must. But we can remove it to clear up the example. Remove?

Reply:
> It's for absolute paths like `import Some from 'src/some.ts'`. I think it's a must. But we can remove it to make the example more clean. Remove?

I guess this is more of a preference thing - to me its sole use is to remove relative imports and some might not want that functionality. I would prefer to stick to the defaults for an example project.